src/holofuel/components/SideMenu/SideMenu.js

```diff
@@ -62,15 +59,6 @@ export default function SideMenu ({
               Profile
             </Link>
           </li>
-          <li>
-            <div styleName='loading-row'>
```

Comment:
The 'loading-row' styles will need to be applied to the previous page link (...and probably renamed), in order to maintain the divider style, separating the holofuel pages from the return to hp-admin link.